### PR TITLE
update 1.0.2 links to 1.2

### PR DESCRIPTION
**Pull request by mystiberry-okta**
@Ujjwalreddy-okta can you review and okay the changes that you requested? OKTA-87063
(Is UP the right Team?)

### CODE REVIEW

File: _source/docs/guides/okta_sign-in_widget.md [hunk 1]
```diff
@@ -97,11 +97,11 @@ and easily set up a fully featured login experience:
 Here is what is happening in the HTML above: 
 
 First, in the `<head>`
-tag, we include the  `okta-sign-in-1.0.2.min.js` and
-`okta-sign-in-1.0.2.min.css` files. These files have all of the
+tag, we include the  `okta-sign-in-1.2.min.js` and
+`okta-sign-in-1.2.min.css` files. These files have all of the
```
Comment (Ujjwalreddy-okta) [inline _source/docs/guides/okta_sign-in_widget.md]: Shouldn't these be - `okta-sign-in-1.2.0.min.js` and `okta-sign-in-1.2.0.min.css` (1.2.0 vs. 1.2)?

File: _source/docs/guides/okta_sign-in_widget.md [hunk 2]
```diff
@@ -97,11 +97,11 @@ and easily set up a fully featured login experience:
...
 logic for the Okta Sign-In Widget. 
 
-We also include the `okta-theme-1.0.2.css` file, which contains all
+We also include the `okta-theme-1.2.css` file, which contains all
```
Comment (Ujjwalreddy-okta) [inline _source/docs/guides/okta_sign-in_widget.md]: Same as above

File: _source/docs/guides/okta_sign-in_widget.md [hunk 3]
```diff
@@ -355,9 +355,9 @@ is up to you. Customization of the widget itself will be done on
 the `#okta-login-container` selector and its child elements.
 
 A full list of the CSS selectors that you can use to style the
-Okta Sign-In Widget are in the [okta-theme.css](https://example.okta.com/js/sdk/okta-theme-1.0.2.css) file. We strongly
+Okta Sign-In Widget are in the [okta-theme.css](https://example.okta.com/js/sdk/okta-theme-1.2.css) file. We strongly
 urge you to style your widget using only the selectors that are
-present in the [okta-theme.css](https://example.okta.com/js/sdk/okta-theme-1.0.2.css) file, other elements in the widget
+present in the [okta-theme.css](https://example.okta.com/js/sdk/okta-theme-1.2.css) file, other elements in the widget
 will be subject to change and will cause your styling to break in
 future versions of the Okta Sign-In Widget.
```
Comment (Ujjwalreddy-okta) [inline _source/docs/guides/okta_sign-in_widget.md]: https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/1.2.0/css/okta-theme-1.2.0.css
in place of -
https://example.okta.com/js/sdk/okta-theme-1.2.css

**Comment by Ujjwalreddy-okta** [comment]
I think we should replace 1.2 with 1.2.0 just to be consistent and avoid confusion? 
And yep, UP is the right team :)

**Comment by mystiberry-okta** [comment]
@Ujjwalreddy-okta sorry about the multiple commits, but I think it's all fixed now.

**Comment by Ujjwalreddy-okta** [comment]
Awesome, looks good. Thanks.
:ship: :rocket:

**Pull request merged**